### ISSUE TITLE
Update to latest client library

### ISSUE THREAD

**Issue by alexcombessie** [MeaningCloud/dss-meaningcloud-plugin#2]
Hi @bgalego,

Hope all is well in these complex times.

I saw that you recently released MeaningCloud-python 2.0.0 👍 
Do you want to plan a plugin update this year to leverage this new client library?

I guess that's also the opportunity to increase language coverage and possibly add the new features?

Cheers,

Alex

**Comment by bgalego** [comment]
Hi @alexcombessie!

I hope all is well with you guys too. Complex times indeed 😅 

I believe the relase only changes a dependency with an API that's not included in the Dataiku plugin, so on that account it's not an urgent update. 
I'm tagging @voiser, as he is in charge of it now, and he will be able to confirm, as well as evaluate possible improvements derived from the latest updates in MeaningCloud.

Cheers,
Blanca

**Comment by alexcombessie** [comment]
Okay! So meaningcloud_python 1.2.2 is already pointing to the latest APIs? I was asking as sometimes there is a hard dependency between API client library version and API version (at least for X.0 major releases)

For the additional API, it'd be a nice-to-have. What's the use case?

**Comment by bgalego** [comment]
That's correct, meaningcloud_python 1.2.2 points to all the latest APIs of the services implemented in the plugin (which are not all the services provided by MeaningCloud). Upgrading the plugin to the new package version should be painless, but as you mentioned, I think it probably makes sense to take the opportunity to add any other changes/improvements.
